UIKitHomeworks/UIKitHomeworks/Register/Model/AuthModel.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  AuthModel.swift
+//  UIKitHomeworks
+//
+//  Created by Vlad on 06.07.2021.
+//
+
+import Foundation
+
+struct User {
+    var name: String
+    var age: String
+    var nickname: String
+    var registrationParameter: RegistationParameter
+    let id: Int
+}
+
+enum RegistationParameter {
+    case phoneNumber(String)
+    case emailAddress(String)
+}
+
+struct AuthModel {
```

Comment:
```suggestion
struct Auth {
```
Обычно модель не называют Model)